ls1intum/Artemis
# [Documentation] Updating Exam Instructor Guide

### Checklist

### Motivation and Context
This PR aims to update the instructor exam guide. @PatZintel updates the student exam guide in another PR. 

### Description
I have updated the instructor exam documentation

### Steps for Testing
Read the instructor exam documentation and make sure that every screenshot provides an accurate representation of the current system.
You can download the documentation here: https://github.com/ls1intum/Artemis/actions/runs/1027593100
